src/main/java/uk/gov/hmcts/cmc/claimstore/controllers/IntegrationTestSupportController.java

```diff
@@ -57,9 +64,24 @@ public Claim updateRespondByDate(
         return getClaim(claimReferenceNumber, authorisation);
     }
 
+    @PutMapping("/claims/{claimReferenceNumber}/defendant/{defendantUsername}")
+    public Claim linkDefendantToClaim(
+        @PathVariable("claimReferenceNumber") String claimReferenceNumber,
+        @PathVariable("defendantUsername") String defendantUsername,
+        @RequestHeader(value = HttpHeaders.AUTHORIZATION, required = false) String authorisation
+    ) {
+        Claim claim = getClaim(claimReferenceNumber, authorisation);
+
+        //todo how we should store default password? const? properties?
+        User defendant = this.userService.authenticateUser(defendantUsername, "Password12");
+
+        supportRepository.linkDefendantToClaim(claim, defendant);
+
+        return getClaim(claimReferenceNumber, authorisation);
```

Comment:
We do not have to return claim back. Can we please remove it as we want to stop doing it in CCD world. If there is a need for frontend to retrieve claim there will be explicit call.